### ISSUE TITLE
feature/SGDRegressor and SGDClassifier should support configurable loss functions

### ISSUE THREAD

**Issue by JasonShin** [kalimdorjs/kalimdorjs#89]
* **I'm submitting a ...**
[/] feature request

* **Summary**

Reference: http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html

Source code location: https://github.com/JasonShin/kalimdorjs/blob/master/src/lib/linear_model/stochastic_gradient.ts#L202

**Comment by JasonShin** [comment]
This is done in https://github.com/kalimdorjs/kalimdorjs/pull/119
